### ISSUE TITLE
LV power generators

### ISSUE THREAD

**Issue by Dreminon** [GTNewHorizons/NewHorizons#1190]
We at prospercraft recently setup a new server for people who wished to play your pack agin. I started it and got full into LV getting ready for MV. so I setup a pump with a refinery and a few diesel gens,like i had before then found you could not make regular fuel had to make diffrent kinds, no problem, the diesel gens run on light sulfur fuel, but upon setting them up to run figured out the generators was not even able to make enough fuel to power the machines themself, let alone to be able to make extra energy. so then I went to try biofuel yes i was aware that the recipe had changed somewhat by you adding another machine needed to make it i set it up with the compressor feeding into the maceator, mixer with water , brewery with water, distillery with circit set to 4 (since it is LV) but once agin it was unable to make enough fuel to run 2 generators which is what is needed to power all the machines and to make just a little extra power.
in short you have basicly made the LV choice for power only Steam. I think you guys need to go over the fuel ratios and tweek them just a bit for low voltage so that oil and biofuel will at least be fesible to use.

**Comment by Dreminon** [comment]
It  end up costing you energy to run both the oil and biosetups.

**Comment by kokochicken** [comment]
This is not true. Are you using light fuel or sulfuric light fuel? And as well the biogas recipe makes a very small profit in lv because it is meant to need the mv distillery

**Comment by Dreminon** [comment]
useing sulfuric light fuel and the biogas has no return. not alittle

**Comment by DFliyerz** [comment]
You can make normal light fuel at LV

**Comment by Dreminon** [comment]
ok ill try the light fuel for oil but my biogas setup still is useless.

**Comment by DietmarKracht** [comment]
Biogas will produce  more energy than  consume

**Comment by Dream-Master** [comment]
@Dreminon 
Can you make some examples with recipe costs and fuel values that i know where i have to buff stuff.

**Comment by Dream-Master** [comment]
Ok after chatting with @Dreminon the problem was solved.
